library/core/src/sync/atomic.rs

```diff
@@ -428,7 +465,8 @@ unsafe impl<T> Sync for AtomicPtr<T> {}
 ///
 /// [nomicon]: ../../../nomicon/atomics.html
 #[stable(feature = "rust1", since = "1.0.0")]
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+#[cfg_attr(not(feature = "ferrocene_certified"), derive(Debug, Eq, PartialEq, Hash))]
+#[derive(Copy, Clone)]
```

Comment:
Here I am not sure if it has a benefit to leave the original derive untouched, since we are inserting the cfg_attr anyways. What do you think is better to avoid merge conflicts, the way you have it now or:

```suggestion
#[cfg_attr(not(feature = "ferrocene_certified"), derive(Copy, Clone, Debug, Eq, PartialEq, Hash))]
#[cfg_attr(feature = "ferrocene_certified", derive(Copy, Clone))]
```